frame/staking/src/lib.rs

```diff
@@ -159,7 +159,7 @@
 //! ```
 //! use pallet_staking::{self as staking};
 //!
-//! #[frame_support::pallet]
+//! #[frame_support::pallet(dev_mode)]
```

Comment:
I guess you are trying to get rid of warnings in the doc tests? 😄

Reply:
Yes :D